src/__test__/Components.test.js

```diff
@@ -0,0 +1,54 @@
+import { render } from '@testing-library/react';
+import { StrictMode } from 'react';
+import { Provider } from 'react-redux';
+import { BrowserRouter as Router } from 'react-router-dom';
+import store from '../redux/configureStore';
+import Home from '../pages/Home';
+import Contact from '../pages/Contact';
+import Details from '../pages/Details';
+import '@testing-library/jest-dom';
+
```

Comment:
- [ ] It would be better if we add more test to our app, for example; we can perform a mocking data fetching :man_technologist: